src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.model;
+
+import java.util.Objects;
+
+public class Fruit {
+    private String fruitName;
+    private int quantity;
```

Comment:
quantity is not a Fruit characteristic. Fruit can not know its quantity. Storage knows it.
Actually we do not need Fruit class in this task because String fruitName is the only thing that we know about Fruit.
We can use Map<String, Integer> as Fruit Storage